docs/python_armada_client.md

```diff
@@ -416,3 +460,61 @@ Definition can be found at:
 * **Parameters**
 
     **event** (*EventStreamMessage*) – The gRPC proto event
+
+
+## armada_client.permissions module
+
+
+### _class_ armada_client.permissions.Permissions(subjects, verbs)
+Permissions including Subjects and Verbs
+
+
+* **Parameters**
+
+    
+    * **subjects** (*List**[**Subject**]*) – 
+
+
+    * **verbs** (*List**[**str**]*) – 
+
+
+
+#### to_grpc()
+Convert to grpc object
+
+
+* **Return type**
+
+    *Permissions*
+
+
+
+### _class_ armada_client.permissions.Subject(kind, name)
+Subject is a NamedTuple that represents a subject in the permission system.
+
+
+* **Parameters**
+
+    
+    * **kind** (*str*) – 
+
+
+    * **name** (*str*) – 
+
+
+
+#### kind(_: st_ )
```

Comment:
@jayofdoom any suggestions on how to try and remove the duplicate documentaition of these types?

Reply:
Lets merge this as-is, and tackle this separately. I have a few ideas but no need to hold up this PR on it.